data-serving/data-service/test/util/search.test.ts

```diff
@@ -0,0 +1,18 @@
+import parseSearchQuery from '../../src/util/search';
+
+describe('search query', () => {
```

Comment:
All of these tests don't many any changes to the query provided. Could you add a test where it does?

Reply:
sorry can you say that again? I think there's a typo and I don't understand what needs to change sorry

Reply:
I was saying that all the tests do is take the input string and put it in an object e.g. `'some query'` -> `{ fullTextSearch: 'some query' }`
But I now see that is all search.ts does for now until the tokens are added, so no updates necessary.